Source/WebCore/dom/ContainerNode.cpp

```diff
@@ -827,12 +827,26 @@ void ContainerNode::removeChildren()
 
     Ref protectedThis { *this };
     NodeVector removedChildren;
-    removeAllChildrenWithScriptAssertion(ChildChange::Source::API, removedChildren);
-
+#if PLATFORM(MAC)
```

Comment:
Why do we need this #if? opportunisticSweepingAndGarbageCollection is false in iOS so the code below will work in iOS as well as macOS.

Reply:
It's for GTK platform. We should probable use `#if PLATFORM(COCOA)` instead.

Reply:
But OpportunisticSweepingAndGarbageCollectionEnabled flag exists everywhere and it's off by default.